packages/paste-website/src/pages/inclusive-design/index.mdx

```diff
@@ -0,0 +1,419 @@
+        <ListItem>Match semantics to behavior for assistive technology users</ListItem>
+        <ListItem>Ensure disabled controls are not focusable</ListItem>
+        <ListItem>
+          Provide a skip link if necessary{' '}
+          <Anchor showExternal href="https://webaim.org/techniques/skipnav/">(Read more about why this matters)</Anchor>
+        </ListItem>        
+        <ListItem>Make sure controls within hidden content are not focusable</ListItem>
+      </UnorderedList>
+    </DisclosureContent>
+  </Disclosure>
+  <Disclosure>
+    <DisclosureHeading as="h3" variant="heading30">
+      Other considerations
+    </DisclosureHeading>
+    <DisclosureContent>
+      <UnorderedList>
+        <ListItem>
+          Make sure main body (paragraph) text is no smaller than the default (user agent) size{' '}
+          <Anchor showExternal href="https://www.smashingmagazine.com/2011/10/16-pixels-body-copy-anything-less-costly-mistake/">
+            (Read more about this)
+          </Anchor>
+        </ListItem>
+        <ListItem>Use relative units (`em`, `rem`, and `ch`), especially for font metrics</ListItem>
+        <ListItem>
+          Do not include third parties that compromise user privacy{' '}
+          <Anchor showExternal href="https://css-tricks.com/potential-dangers-of-third-party-javascript/">(Read more about why this matters)</Anchor>
+        </ListItem>
+        <ListItem>
+          Provide `title`s that name the site and the specific page{' '}
+          <Anchor showExternal href="https://www.w3.org/TR/WCAG20-TECHS/G88.html">(Read more about why this matters)</Anchor>
+        </ListItem>
+        <ListItem>
+          Honour DNT (Do Not Track) header{' '}
+          <Anchor showExternal href="https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/DNT">(Read more about why this matters)</Anchor>
+        </ListItem>
+        <ListItem>
+          Provide a default language and use `lang="[ISO code]"` for subsections in different languages
+        </ListItem>
+        <ListItem>
+          Make sure all content belongs to a landmark element (`header`, `footer`, `nav`, `main`, etc)
+        </ListItem>
+        <ListItem>Provide a `manifest.json` file for identifiable homescreen entries</ListItem>
+        <ListItem>Make sure data tables wider than their container can be scrolled horizontally</ListItem>
+        <ListItem>
+          Make content easier to find and improve search results with structured data{' '}
+          <Anchor showExternal href="https://developers.google.com/search/docs/guides/prototype">(Read more about why this matters)</Anchor>
+        </ListItem>
+        <ListItem>Provide a print stylesheet (single column, with interactive content hidden)</ListItem>
+        <ListItem>
+          Instead of obstructing users with CAPTCHAs, use honeypots{' '}
+          <Anchor showExternal href="<https://en.wikipedia.org/wiki/Honeypot_(computing)>">(Read more about why this matters)</Anchor>
```

Comment:
Link is broken, gotta remove the `<` and `>`.  I also like this page more: https://www.w3.org/WAI/GL/wiki/Captcha_Alternatives_and_thoughts since the one linked is just about general honeypots in computing, which has different implications based on environment.